posts/vue-svg-symbol-sprites.md

```diff
@@ -0,0 +1,177 @@
+---
+title: Using SVG symbol sprites with Vue
+date: 2019-02-22
+---
+
+I recently compared the most popular methods of embedding icons in web application and came to the conclusion, that SVG symbols are probably the best solution in 2019.
+
+Other options are:
+
+* Icon Fonts, which usually work fine but are a bit difficult to maintain because you have to regenerate binary font files every time you add or change an icon. Also in my experience they sometimes fail to load in mysterious ways.
+* Inline SVGs are convenient to implement and work with but can't be reused within a project and blow up the HTML, which can be annoying during development and can also hurt performance in the browser, especially when being repeated in long lists.
```

Comment:
Inline SVGs are convenient to implement and **to** work with but can't be reused within a project and blow up the HTML**. This** can be annoying during development and can also hurt performance in the browser, especially when being repeated in long lists.